content/fr/docs/concepts/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Concepts
+main_menu: true
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+La section Concepts vous aide à mieux comprendre les composants du système Kubernetes et ses abstractions qu'il utilise pour représenter votre cluster. Elle vous aide également à mieux comprendre le fonctionnement de Kubernetes.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Vue d'ensemble
+
+Pour utiliser Kubernetes, vous utilisez *les objets de l'API Kubernetes* pour décrire l'état souhaité *de votre cluster*: quelles applications ou autres charges de travail que vous souhaitez exécuter, quelles images de conteneur elles utilisent, le nombre de réplicas, les ressources réseau et disque souhaitées mettre à disposition, et plus encore. Vous définissez l'état souhaité en créant des objets à l'aide de l'API Kubernetes, généralement via l'interface de ligne de commande, `kubectl`. Vous pouvez également utiliser l'API Kubernetes directement pour interagir avec le cluster et définir ou modifier l'état souhaité.
```

Comment:
add "," after *de votre cluster*
switch "charges de travail" with: "workloads (charge de travail)" as we discuss yesterday
...et disques **que vous souhaitez** mettre a disposition...